src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -343,17 +343,14 @@ private DatabaseConnectionState GetOrAddConnectionState(string name, string watc
             catch (OperationCanceledException e)
             {
                 NotifyAboutError(e);
+                _tcs.TrySetException(e);
```

Comment:
Shouldn't this be `TrySetCancelled`?

Reply:
I'm not sure, It's possible that we want to track the exception stack trace, or we just want to know that it was canceled?

Reply:
I think we should go with `TrySetCanceled`. You will not get much more details here anyway.